README.md

```diff
@@ -7,6 +7,43 @@
 [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7024/badge)](https://bestpractices.coreinfrastructure.org/projects/7024)
 [![Discussions](https://img.shields.io/github/discussions/OWASP/wrongsecrets)](https://github.com/OWASP/wrongsecrets/discussions)
 
+### Table of contents
```

Comment:
Can you move the TOC to just above support please? So people can read what it is immediately:-)

Reply:
above the heading?

Reply:
Yes please :-) above the heading

Reply:
Ok sir i'll do it in a jiffy....